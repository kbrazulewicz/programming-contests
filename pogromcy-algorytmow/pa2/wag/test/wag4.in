204
282
141
188
282
47
141
329
141
141
94
94
94
47
188
141
188
188
235
47
94
188
235
282
94
47
235
235
235
329
282
141
188
94
188
188
188
94
47
329
94
188
94
94
94
94
282
329
282
235
329
329
329
94
235
141
141
188
94
282
235
329
47
141
282
141
47
94
282
235
47
47
329
141
94
141
282
94
141
329
188
47
235
47
141
235
329
329
47
235
141
329
94
188
329
188
94
282
141
329
329
329
282
235
188
329
188
94
282
329
94
329
329
94
141
94
94
47
329
188
141
141
329
94
94
188
329
188
47
141
188
235
47
47
47
47
47
329
47
282
282
235
329
329
188
94
141
94
94
94
235
282
235
94
329
94
282
235
235
329
235
141
47
188
329
282
329
47
94
235
141
94
188
235
188
188
94
94
47
94
282
329
47
235
235
188
94
141
188
235
282
282
47
282
235
94
235
47
141
188
141
3115
1882
1274
1781
