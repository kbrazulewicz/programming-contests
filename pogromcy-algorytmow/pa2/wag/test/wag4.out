45 2
94 94 94 94 94 94 94 94 94 94 94 94 94 94 94 94 94 94 94 94 47 47 47 47 47 47 47 47 47 47 47 47 47 47 47 47 47 47 47 47 47 47 47 47 47
1781 1274
